pages/1.11/networking/SDN/dcos-overlay/index.md

```diff
@@ -198,3 +226,30 @@ To replace your virtual network, uninstall DC/OS and delete the overlay replicat
 # Troubleshooting
 
 The **Networking** tab of the DC/OS web interface provides information helpful for troubleshooting. You can see which containers are on which network and see their IP addresses.
+
+<a name="limitations"></a>
+### Limitations
+
+* DC/OS Overlay does not allow services to reserve IP addresses that result in ephemeral addresses for containers across multiple incarnations on the virtual network. This restriction ensures that a given client connects to the correct service.
+
+  [VIPs (virtual IP addresses)](/pages/1.11/networking/load-balancing-vips/) are built in to DC/OS and offer a clean way of allocating static addresses to services. If you are using DC/OS Overlay, you should use VIPs to access your services to support cached DNS requests and static IP addresses.
```

Comment:
@asridharan named vips don't have static IP addresses, I think we should remove this paragraph